### PR TITLE
Add mention of patchfile

### PR DESCRIPTION
**Pull request by crutchcorn**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- What does this PR change? Give us a brief description.

This PR introduces mention of the `.patch` trick to lookup a commit author's name and email.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -103,4 +103,10 @@ Typically, this will be an approval from another maintainer, often the Docs Lead
 
 Lastly, we have a "DocsBot" that posts congratulation messages in the #docs Discord channel. **Our bot also acknowledges co-authors!** During "Squash and Merge," if you would like to acknowledge any reviewer, please type in the exact phrase `Co-authored-by:` and include any missing reviewers or contributors whom you'd like to acknowledge!
 
-TODO: List the steps to get the proper name to use in commit message.
+### Getting `Co-Authored-by` Commit Message Name and Email
+
+In order to ensure that we're using the proper commit co-author's name and email associated with their GitHub account, we can look up that information via a GitHub commit.
+
+From an individual's commit, say `https://github.com/withastro/docs/commit/de11f2f2abf7ef54c874ebe0c85301d9bad36094`, add a `.patch` to the end of the URL.
```
Comment (delucis) [inline MAINTAINERS.md:110]:
```suggestion
From an individual's commit, say `https://github.com/withastro/docs/commit/de11f2f2abf7ef54c874ebe0c85301d9bad36094`, add `.patch` to the end of the URL.
```

**Review by delucis — COMMENTED** [review]
Yay! Thanks for helping out with this @crutchcorn!

I've made one tiny suggestion but this looks good to me 🙌

File: MAINTAINERS.md [hunk 2]
```diff
@@ -103,4 +103,10 @@ Typically, this will be an approval from another maintainer, often the Docs Lead
 
 Lastly, we have a "DocsBot" that posts congratulation messages in the #docs Discord channel. **Our bot also acknowledges co-authors!** During "Squash and Merge," if you would like to acknowledge any reviewer, please type in the exact phrase `Co-authored-by:` and include any missing reviewers or contributors whom you'd like to acknowledge!
 
-TODO: List the steps to get the proper name to use in commit message.
+### Getting `Co-Authored-by` Commit Message Name and Email
+
+In order to ensure that we're using the proper commit co-author's name and email associated with their GitHub account, we can look up that information via a GitHub commit.
```
Comment (sarah11918) [inline MAINTAINERS.md:108]:
```suggestion
For the bot to work properly, you must use the exact co-author's name and email associated with their GitHub account. 

You can look up that information using a GitHub commit they have made from any PR, even one from another repository! Please don't hesitate to take the extra step and include others!
```

Thanks @crutchcorn ! I took the liberty of making this a bit more "threatening" (i.e. "you must follow protocol or it won't work!") and changed the verb tense to match the rest, which is telling "you" how to do things.

**Comment by crutchcorn** [comment]
Suggestions applied!

**Review by delucis — APPROVED** [review]
🥳 Thanks @crutchcorn!

**Pull request merged**